src/App.js

```diff
@@ -176,36 +176,22 @@ const RabbitHolePage = withRouter(
         console.log('TITLE FETCH resp', randomTitleData.items[0].title);
         this.setState({ firstPageTitle: randomTitleData.items[0].title});
```

Comment:
You could move the title into a variable:
```suggestion
        const randomPageTitle = randomTitleData.items[0].title;
        console.log('TITLE FETCH resp', randomPageTitle);
        this.setState({ firstPageTitle: randomPageTitle });
```